cmd/create.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/onkernel/cli/pkg/create"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a new application",
+	Long:  "Commands for creating new Kernel applications",
+	RunE:  runCreateApp,
+}
+
+func init() {
+	createCmd.Flags().StringP("name", "n", "", "Name of the application")
+	createCmd.Flags().StringP("language", "l", "", "Language of the application")
+	createCmd.Flags().StringP("template", "t", "", "Template to use for the application")
+}
+
+func runCreateApp(cmd *cobra.Command, args []string) error {
+	appName, _ := cmd.Flags().GetString("name")
+	language, _ := cmd.Flags().GetString("language")
+	template, _ := cmd.Flags().GetString("template")
+
+	appName, err := create.PromptForAppName(appName)
+	if err != nil {
+		return fmt.Errorf("failed to get app name: %w", err)
+	}
+
+	language, err = create.PromptForLanguage(language)
+	if err != nil {
+		return fmt.Errorf("failed to get language: %w", err)
+	}
+
+	template, err = create.PromptForTemplate(template)
+	if err != nil {
+		return fmt.Errorf("failed to get template: %w", err)
+	}
+
+	// Get absolute path for the app directory
+	appPath, err := filepath.Abs(appName)
+	if err != nil {
+		return fmt.Errorf("failed to resolve app path: %w", err)
+	}
+
+	// TODO: handle overwrite gracefully (prompt user)
+	// Check if directory already exists
+	if _, err := os.Stat(appPath); err == nil {
+		return fmt.Errorf("directory %s already exists", appName)
+	}
+
+	// Create the app directory
+	if err := os.MkdirAll(appPath, 0755); err != nil {
+		return fmt.Errorf("failed to create directory: %w", err)
+	}
+
+	pterm.Println(fmt.Sprintf("\nCreating a new %s %s\n", language, template))
+
+	spinner, _ := pterm.DefaultSpinner.Start("Copying template files...")
+
+	if err := create.CopyTemplateFiles(appPath, language, template); err != nil {
+		spinner.Fail("Failed to copy template files")
+		return fmt.Errorf("failed to copy template files: %w", err)
+	}
+	spinner.Success("✔ TypeScript environment set up successfully")
```

Comment:
This message is hardcoded to TypeScript but Python is also supported. Consider: `fmt.Sprintf("✔ %s environment set up successfully", language)`